src/js/polyfills/webvtt.js

```diff
@@ -502,11 +502,17 @@ function CueStyleBox(window, cue) {
         this.applyStyles({
             top: this.formatStyle(box.top, 'px'),
             bottom: this.formatStyle(box.bottom, 'px'),
-            left: this.formatStyle(box.left, 'px'),
             paddingRight: this.formatStyle(box.right, 'px'),
             height: this.formatStyle(box.height, 'px'),
             transform: ''
         });
+
+        // Apply the left styling if the cue has not been wrapped.
+        if (this.cueDiv.style['white-space'] === 'pre') {
```

Comment:
I think this is supposed to be this.cueDiv.style.whiteSpace now. Might want to see if there's any other references to the old property name hanging around as well.